scte35/state.go

```diff
@@ -28,19 +28,6 @@ import "github.com/Comcast/gots"
 
 const receivedRingLen = 10
 
-type receivedElem struct {
```

Comment:
While I don't agree with the pattern of moving private structs/fields to `doc.go` I'm not going to block the PR for it.